packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/mfaoob/use-flow.md

```diff
@@ -0,0 +1,146 @@
+
+```bash
+curl --request POST \
+  --url https://${yourOktaDomain}/oauth2/default/v1/token \
+  --header 'accept: application/json' \
+  --header 'content-type: application/x-www-form-urlencoded' \
+  --data 'client_id=${client_id}&scope=openid profile&grant_type=password&username=${testuser%40example.com}&password={$userpassword}'
+```
+
+Note the parameters that are passed:
+
+- `client_id` matches the client ID of the native application that you created in the [Set up your app](#set-up-your-app) section. You can find it at the top of your app's **General** tab.
+- `scope` must be at least `openid`. See the **Create Scopes** section of the [Create an authorization server guide](/docs/guides/customize-authz-server/main/#create-scopes).
+- `grant_type` is `password`, indicating that you are using the Resource Owner Password grant type.
+- `username` is the identifier for the user (email).
+- `password` is the password of the matching user.
+
+Not used in this example:
+
+`grant_types_supported` (Optional) is a list of grant types that the client supports. Some clients aren't able to support all grant types. Use this parameter to specify to Okta that the included grant types are all that the client supports so that Okta can return an error if the the provided grant types won't satisfy the access policy.
+
+For more information on these parameters, see the `/token` [endpoint](https://developer.okta.com/docs/api/openapi/okta-oauth/oauth/tag/CustomAS/#tag/CustomAS/operation/tokenCustomAS).
+
+**Response**
+
+Since this is a two factor flow, Okta sends an HTTP 403 error and includes the `mfa_token` in the response. The `mfa_token` is a unique token used for identifying multifactor authentication flows to link the request to the original authentication flow.
+
+```json
+    {
+        "error": "mfa_required",
+        "error_description": "Verify with an additional authenticator to complete the sign-in process.",
+        "mfa_token": "fCRU9lDO0rMHQ_FIADFL"
+    }
+```
+
+### Challenge request
+
+Next, your app sends a request to the authorization server `/challenge` endpoint. Use this endpoint to step up authentication in an already partially authenticated flow:
+
+- Always call this endpoint after the authorization server returns `mfa_required` in response to the `/token` request.
+- Don't use this endpoint with primary factor authentication flows.
+- This endpoint is optional. The client can directly call the `/token` endpoint if it has another way of knowing which factor to authenticate with and can challenge the user for that factor.
```

Comment:
For MFA OOB it is not optional. It is only optional for MFA OTP and you already did not include the endpoint in that flow so I think this line can be removed